src/components/todo/TodoSort.vue

```diff
@@ -61,18 +67,22 @@ import { TodoType } from '../../types/TodoType';
 
 interface Props {
   todos: TodoType[]
+  whichArrow: string
 }
 
 defineProps<Props>()
 
 const emit = defineEmits(['sortTodos', 'reverseTodos'])
 
+const active = ref("")
```

Comment:
Please rename the `active` variable to a more descriptive name. Ex. `activeFilter`